srcpkgs/herbe/patches/Xresources.diff

```diff
@@ -0,0 +1,95 @@
+diff --git a/config.def.h b/config.def.h
```

Comment:
elaborate on reasoning about this patch or mention the source if any

Reply:
I got the patch [from here](https://github.com/dudik/herbe/pull/11) and it was convenient for me to add it so that users can modify the settings from the Xresources file and not compile it from source.

Reply:
reference the origin of that patch in the patch header. nobody reads github